test/common/index.js

```diff
@@ -101,3 +101,10 @@ exports.runTestWithBindingPath = async function(test, buildType) {
     await test(item);
   }
 }
+
+exports.runTestNoBinding = async function(test, buildType) {
```

Comment:
The above two helpers are named with their parameters:

- `runTest(test: (binding) => Promise<void>)`
-  `runTestWithBindingPath(test: (bindingPath) => Promise<void>)`

Maybe we can name this as `runTestWithBuildType` so that we can assume the first parameter of the test function is `buildType`?

Reply:
@legendecas that makes sense to me will update.